fe/model_utils.py

```diff
@@ -53,12 +54,24 @@ def is_hydrogen(i):
     return masses
 
 
-# def generate_openmm_topology(objs, host_coords, out_filename):
-def generate_openmm_topology(objs, coords, out_filename=None):
+def image_molecule(mol_coords, box):
+    """Given a set of coordinates for a single molecule and a box will return
+    the coordinates wrapped into the periodic box.
+    """
+    centroid = np.mean(mol_coords, axis=0)
+    box_diag = np.diag(box)
+    new_center = box_diag * np.floor(centroid / box_diag)
+    return mol_coords - new_center
```

Comment:
I might be missing a subtlety, but this seems equivalent to using the `%` operator elementwise. Would the following work?
```suggestion
    return centroid % box_diag
```

Reply:
Ah, never mind, I think I was mixing up `centroid` and `mol_coords`. My suggestion doesn't make sense.

Reply:
`centroid % box_diag` returns the desired center for the molecule after moved, while subtracting the current center actually places the coordinates in the box so that they are centered correctly. Will expand the test to make that a bit more clear, as having it as a single set of coordinates allows this to be a reasonable way to do it.